corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -225,9 +238,12 @@ public int loadTable(String namespace, String tablename, int numItems, int batch
                     TableName.getDefaultInstance().getClass(),
                     optionsBuilder.build());
 
-            TableName dummyVal = TableName.newBuilder().setNamespace(namespace).setTableName(tablename).build();
-            log.info("Loading {} items in {} batchSized transactions into {}${}",
-                    numItems, batchSize, namespace, tablename);
+            byte[] array = new byte[itemSize];
+            new Random().nextBytes(array);
+            TableName dummyVal = TableName.newBuilder().setNamespace(namespace+tablename)
```

Comment:
why namespace+tablename?

Reply:
just to display something meaningful in case someone is silly enough to dump the dummy table :)